code/game/gamemodes/game_mode.dm

```diff
@@ -400,15 +400,19 @@ proc/display_roundstart_logout_report()
 			to_chat(M, msg)
 
 
-proc/get_nt_opposed()
+/proc/get_nt_opposed()
 	var/list/dudes = list()
 	for(var/mob/living/carbon/human/man in player_list)
 		if(man.client)
+			//don't try picking someone like the captain or a security officer for potential collaborators, even if they ARE opposed to Nanotrasen for some reason

```

Comment:
Don't know how I feel about an officer being kept out. I like the idea of a bad cop in the force but I can see it being a problem either way